src/test/regress/sql/multi_mx_call.sql

```diff
@@ -0,0 +1,35 @@
+-- Test passing off CALL to mx workers
+
+-- Create worker-local tables to test procedure calls were routed
+
+set citus.shard_replication_factor to 1;
+
+SELECT run_command_on_workers($$
+create table mx_call_table (val int);
+insert into mx_call_table values (2);
+$$);
+
+CREATE TABLE mx_call_dist_table(id int);
+select create_distributed_table('mx_call_dist_table', 'id');
+
+CREATE PROCEDURE mx_call_proc(x int, INOUT y int) LANGUAGE plpgsql AS $$
+BEGIN
+    y := x + (select val from mx_call_table);
```

Comment:
You could use the `groupid` value in `pg_dist_local_group` to distinguish workers.